routing/routing_session.cpp

```diff
@@ -523,6 +523,26 @@ void RoutingSession::EmitCloseRoutingEvent() const
       alohalytics::Location::FromLatLon(lastGoodPoint.lat, lastGoodPoint.lon));
 }
 
+bool RoutingSession::HasRouteAltitudeImpl() const
+{
+  return !m_route.GetAltitudes().empty();
+}
+
+bool RoutingSession::HasRouteAltitude() const
+{
+  threads::MutexGuard guard(m_routeSessionMutex);
+  return HasRouteAltitudeImpl();
+}
+
+bool RoutingSession::CopyRouteAltitudes(feature::TAltitudes & routeAltitudes) const
```

Comment:
Getters may not be lightweight. Moreover, the method signature indicates that it's not lightweight.